### ISSUE TITLE
[Bug]: Stack on Loading stages on latest update  

### ISSUE THREAD

**Issue by Azizadx** [httpie/desktop#105]
### What happened?

As you can see below my app stack is on loading stages. I don't know why
![Screenshot 2023-09-06 at 6 44 44 PM](https://github.com/httpie/desktop/assets/45791956/f271f3f8-d5ef-40d9-b46e-c6f5a40a1d8c)

### Steps to reproduce?

```Markdown
1. I launch the app
2. Then a pop dialogue asks me if I want to update the software
3. After the installation is done
4. I tried to relaunch the app it stacks on the loading stage
```

### What did you expect to happen?

when the update process is done i want to relaunch the app and use the new version

### Platform

macOS

### HTTPie Desktop version

2022.5.1

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Azizadx** [comment]
**@jkbrzt sure there you go**

index.js:184 Refused to create a worker from 'blob:file:///24c96d66-bca3-417c-8027-3b3dbcba63cf' because it violates the following Content Security Policy directive: "script-src 'self' data: 'unsafe-eval' 'unsafe-inline'". Note that 'worker-src' was not explicitly set, so 'script-src' is used as a fallback.

eventBuffer.useCompression @ index.js:184
index.js:184 Object
index.js:184 Objectcache: e {models: Array(11), collections: Map(8)}[[Prototype]]: Object
index.js:184  Starting as non-leader tab
index.js:184  Leader tab elected
index.js:184 updateManager.start
index.js:184 updateManager.checkForUpdates
index.js:184 realtime.start
index.js:184 service:mandatoryRecordsManager Ensuring profile exists
index.js:184 service:mandatoryRecordsManager Ensuring space exists
index.js:184 service:mandatoryRecordsManager Ensuring application state exists
index.js:184 service:mandatoryRecordsManager Ensuring application window exists
index.js:184 AuthStatus: AUTHENTICATED
index.js:184 updateManager.stats UP-TO-DATE → UP-TO-DATE
index.js:184 updateManager.setIsUrgent false
index.js:184 realtime.onConnect
index.js:184 sync.error null → null
index.js:184 updateManager.setIsUrgent false
index.js:184   sync.status@profiles IDLE → SYNCING
index.js:184 sync.status IDLE → SYNCING
index.js:184   sync.status@profiles SYNCING → IDLE
index.js:184 sync.status SYNCING → IDLE
index.js:184   sync.status@workspaces IDLE → SYNCING
index.js:184 sync.status IDLE → SYNCING
index.js:184   sync.status@workspaces SYNCING → IDLE
index.js:184 sync.status SYNCING → IDLE
index.js:184   sync.status@environments IDLE → SYNCING
index.js:184 sync.status IDLE → SYNCING
index.js:184   sync.status@environments SYNCING → IDLE
index.js:184 sync.status SYNCING → IDLE
index.js:184   sync.status@variables IDLE → SYNCING
index.js:184 sync.status IDLE → SYNCING
index.js:184   sync.status@variables SYNCING → IDLE
index.js:184 sync.status SYNCING → IDLE
index.js:184   sync.status@variablevalues IDLE → SYNCING
index.js:184 sync.status IDLE → SYNCING
index.js:184   sync.status@variablevalues SYNCING → IDLE
index.js:184 sync.status SYNCING → IDLE
index.js:184   sync.status@collections IDLE → SYNCING
index.js:184 sync.status IDLE → SYNCING
index.js:184   sync.status@collections SYNCING → IDLE
index.js:184 sync.status SYNCING → IDLE
index.js:184   sync.status@requestspecs IDLE → SYNCING
index.js:184 sync.status IDLE → SYNCING
index.js:184   sync.status@requestspecs SYNCING → IDLE
index.js:184 sync.status SYNCING → IDLE
DevTools failed to load source map: Could not parse content for file:///Applications/HTTPie.app/Contents/Resources/app.asar/.webpack/renderer/index.js.map: Unexpected end of JSON input
DevTools failed to load source map: Could not parse content for file:///Applications/HTTPie.app/Contents/Resources/app.asar/.webpack/renderer/464.index.js.map: Unexpected end of JSON input
DevTools failed to load source map: Could not parse content for file:///Applications/HTTPie.app/Contents/Resources/app.asar/.webpack/renderer/493.index.js.map: Unexpected end of JSON input
DevTools failed to load source map: Could not parse content for file:///Applications/HTTPie.app/Contents/Resources/app.asar/.webpack/renderer/214.index.js.map: Unexpected end of JSON input
DevTools failed to load source map: Could not parse content for file:///Applications/HTTPie.app/Contents/Resources/app.asar/.webpack/renderer/main.css.map: Unexpected end of JSON input

**Comment by jkbrzt** [comment]
Something might be missing. Could you please restart the app, let it sit for a bit, and then copy/paste the console output again?

**Comment by Azizadx** [comment]
I've already but sure np I'll

**Comment by jkbrzt** [comment]
@Azizadx it might be hard to debug this without access to the database. Would you mind zipping the following folder:

```
~/Library/Application Support/HTTPie
```

and sending it to me at jakub@httpie.io?

**Comment by jkbrzt** [comment]
Great, thanks for sending the file. We’re looking into it. If all your data is synced to the cloud, you can quit the app, delete the directory, start the app and log in again. That might make the problem go away. We’ll be working on a proper fix in the meantime.

**Comment by Azizadx** [comment]
That could be the only reason I do not want to delete it. But I'll figure it out.
I'll close this issue for the time being in case anything happens I'll reopen it. 

**Thanks for the fast response I appreciate 🫡.**

**Comment by jkbrzt** [comment]
@Azizadx let’s keep it open for now; we’ll close it once the issue has been solved

**Comment by jkbrzt** [comment]
@Azizadx here’s a way how to recover from this type of errors:

1. Open DevTools (Shift-Option-I)
2. Detele the `windows-0.chunk.0` key from IndexedDb as indicated below:

<img width="818" alt="Screenshot 2023-10-04 at 4 53 10 PM" src="https://github.com/httpie/desktop/assets/326885/f062a083-f383-444b-8a34-5aa765d741a0">

A proper fix coming 🔜

**Comment by jkbrzt** [comment]
Fixed in [v2023.3.6](https://github.com/httpie/desktop/releases/tag/) 🥧

**Comment by Azizadx** [comment]
> @Azizadx here’s how to recover from this type of error:
> 
> 1. Open DevTools (Shift-Option-I)
> 2. Delete the `windows-0.chunk.0` key from IndexedDb as indicated below
> 3. Restart the app
> 
> <img alt="Screenshot 2023-10-04 at 4 53 10 PM" width="818" src="https://user-images.githubusercontent.com/326885/272618544-f062a083-f383-444b-8a34-5aa765d741a0.png">
> A proper fix is coming 🔜

@jkbrzt This work finally sorry for my late response
